server/api/internal/infrastructure/mongo/mongodoc/worker_config.go

```diff
@@ -0,0 +1,70 @@
+package mongodoc
+
+import (
+	"time"
+
+	"github.com/reearth/reearth-flow/api/pkg/batchconfig"
+	"github.com/reearth/reearth-flow/api/pkg/id"
+)
+
+type WorkerConfigDocument struct {
+	Workspace             string    `bson:"workspace"`
+	MachineType           *string   `bson:"machine_type,omitempty"`
+	ComputeCpuMilli       *int      `bson:"compute_cpu_milli,omitempty"`
+	ComputeMemoryMib      *int      `bson:"compute_memory_mib,omitempty"`
+	BootDiskSizeGB        *int      `bson:"boot_disk_size_gb,omitempty"`
+	TaskCount             *int      `bson:"task_count,omitempty"`
+	MaxConcurrency        *int      `bson:"max_concurrency,omitempty"`
+	ThreadPoolSize        *int      `bson:"thread_pool_size,omitempty"`
+	ChannelBufferSize     *int      `bson:"channel_buffer_size,omitempty"`
+	FeatureFlushThreshold *int      `bson:"feature_flush_threshold,omitempty"`
+	NodeStatusDelayMilli  *int      `bson:"node_status_delay_ms,omitempty"`
```

Comment:
Field name is NodeStatusDelayMilli but BSON tag is node_status_delay_ms. this naming is inconsitent